File: samples/go/01table/table.go
package main

import (
    "fmt"
)


func main() {
    const LIMIT int = 5;
    fmt.Printf("   ");
    for i := 1; i <= LIMIT; i++ {
        fmt.Printf("%-3d", i);
    }
    fmt.Println();

    for a := 1; a <= LIMIT; a++ {
        fmt.Printf("%-3d", a);
        for b := 1; b <= LIMIT; b++ {
            fmt.Printf("%-3d", a*b);
        }
        fmt.Println();
    }
}

File: samples/python/01table/table.py
import sys


LIMIT = 5


def main():
    for a in range(LIMIT+1):
        if a:
            sys.stdout.write("%-3d" % (a,))
        else:
            sys.stdout.write("%-3s" % ("",))
        for b in range(LIMIT+1):
            if b:
                if a:
                    sys.stdout.write("%-3d" % (a*b,))
                else:
                    sys.stdout.write("%-3d" % (b,))
        print()


if __name__ == "__main__":
    main()
